posthog/utils.py

```diff
@@ -1042,6 +1041,15 @@ def cache_requested_by_client(request: Request) -> bool | str:
     return _request_has_key_set("use_cache", request)
 
 
+def filters_override_requested_by_client(request: Request) -> Optional[dict]:
+    raw_filters = request.query_params.get("filters_override")
+
+    if raw_filters is not None:
+        return json.loads(raw_filters)
```

Comment:
This might result in a JSON parsing error – in that case we should catch it and throw a ValidationError instead to avoid a 500